bagitobjecttransfer/recordtransfer/templates/recordtransfer/profile.html

```diff
@@ -1,145 +1,177 @@
-    {% csrf_token %}
-    {% include 'recordtransfer/messages.html' %}
-    <div class="flex-item margin-top-25px">
-        <label>{% trans "Name" %}</label>
-        <input disabled value="{{ user.get_full_name }}">
-    </div>
-    <div class="flex-item">
-        <label>{% trans "Email" %}</label>
-        <input disabled value="{{ user.email }}">
-    </div>
-    <div class="flex-item">
-        <label>{% trans "Username" %}</label>
-        <input disabled value="{{ user.username }}">
-    </div>
-    <div class="flex-item">
-        <label for="{{ form.gets_notification_emails.id_for_label }}">{% trans "Receive notification emails?" %}</label>
-        {{ form.gets_notification_emails }}
-    </div>
-    <div class="flex-item">
-        <label></label>
-        <button type="submit" class="blue-button">
-            {% trans "Save" %}
-        </button>
-    </div>
-    </form>
-
-    <br>
-
-    {% if in_process_submissions %}
-        <div class="flex-med-width-item margin-top-25px" id="ongoing-transfers">
-            <label class="medium-text">{% trans "Ongoing Transfers"%}</label>
-        </div>
-        <div class="flex-med-width-item">
-            <table class="full-width-table standard-table" cellspacing="0">
-                <tr>
-                    <th>{% trans "Last Updated" %}</th>
-                    <th></th>
-                    <th></th>
-                </tr>
-                {% for in_process in in_process_submissions %}
-                <tr>
-                    <td>{{ in_process.last_updated|date:"D M j, Y" }} @ {{ in_process.last_updated|time:"H:i" }}</td>
-                    <td><a href="{% url 'recordtransfer:transfer' %}?resume_transfer={{ in_process.id }}">Resume</a></td>
-                    <td><a href="{% url 'recordtransfer:transferdelete' in_process.id %}">Delete</a></td>
-                </tr>
-                {% endfor %}
-            </table>
+    {% if user.is_authenticated %}
+        {% include "recordtransfer/messages.html" %}
+        <div class="title-text">{% trans "Your Profile" %}</div>
+        <form method="post" action="{% url 'recordtransfer:userprofile' %}">
+            {% csrf_token %}
+            <div class="flex-item margin-top-25px">
+                <label>{% trans "Name" %}</label>
+                <input disabled value="{{ user.get_full_name }}">
+            </div>
+            <div class="flex-item">
+                <label>{% trans "Email" %}</label>
+                <input disabled value="{{ user.email }}">
+            </div>
+            <div class="flex-item">
+                <label>{% trans "Username" %}</label>
+                <input disabled value="{{ user.username }}">
+            </div>
+            <div class="flex-item">
+                <label for="{{ form.gets_notification_emails.id_for_label }}">{% trans "Receive notification emails?" %}</label>
+                {{ form.gets_notification_emails }}
+            </div>
+            <br>
```

Comment:
Instead of a `<br>` can you add a heading that says "Password Reset"? The same medium-text class as "Past Transfers" can be used.

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/208/commits/6842f1408d83a768a44705f2aec5c5a7e3d03798